### DISCUSSION TITLE
Fuzzy matching sensitivity

### DISCUSSION THREAD

**Discussion by tomkolp** [WeblateOrg/weblate#5431] [Ideas] [answered]
Is fuzzy matching configurable in any way?  In my case, it would be better to have a higher compliance threshold.  Is it possible to mark changed strings as "need editing" and new ones without any flag?

**Comment by nijel** [comment, marked as answer]
In case you are using Gettext and msgmerge, I don't think there is a way to configure this.

In case you want to set new strings in Weblate as needing editing, there are several addons to do this: https://docs.weblate.org/en/latest/admin/addons.html#flag-new-translations-as-needs-editing
